src/Adapter/Currency/Repository/CurrencyRepository.php

```diff
@@ -69,4 +69,14 @@ public function get(CurrencyId $currencyId): Currency
 
         return $currency;
     }
+
+    /**
+     * @param CurrencyId $currencyId
+     *
+     * @return string
+     */
+    public function getIsoCode(CurrencyId $currencyId): string
+    {
+        return $this->get($currencyId)->iso_code;
```

Comment:
If it's just to get the iso code, maybe a custom query via doctrine connection would be more efficient than fetching a whole ObjectModel?